maro/cli/grass/lib/services/node_agent/agent.py

```diff
@@ -382,30 +368,29 @@ def load_images(self) -> None:
         for image_file_name, image_file_details in name_to_image_file_details_in_master.items():
             if (
                 image_file_name not in name_to_image_file_details_in_node
-                or name_to_image_file_details_in_node[image_file_name]["md5_checksum"] !=
-                image_file_details["md5_checksum"]
+                or name_to_image_file_details_in_node[image_file_name]["md5_checksum"]
+                != image_file_details["md5_checksum"]
             ):
                 unloaded_image_names.append(image_file_name)
 
         # Parallel load
         with ThreadPool(5) as pool:
             params = [
-                [os.path.expanduser(
-                    f"~/.maro-shared/clusters/{self._local_cluster_details['name']}/image_files/{unloaded_image_name}")]
+                [
+                    os.path.expanduser(
+                        f"~/.maro-shared/clusters/{self._local_cluster_details['name']}/image_files/{unloaded_image_name}"
+                    )
+                ]
                 for unloaded_image_name in unloaded_image_names
             ]
-            pool.starmap(
-                self._load_image,
-                params
-            )
+            pool.starmap(self._load_image, params)
 
         with self._redis_controller.lock(f"lock:name_to_node_details:{self._local_node_details['name']}"):
             node_details = self._redis_controller.get_node_details(node_name=self._local_node_details["name"])
             # Update with mapping in master.
             node_details["image_files"] = name_to_image_file_details_in_master
             self._redis_controller.set_node_details(
-                node_name=self._local_node_details["name"],
-                node_details=node_details
+                node_name=self._local_node_details["name"], node_details=node_details
```

Comment:
Add tailing `,` or not?

Reply:
See [this](https://black.readthedocs.io/en/stable/the_black_code_style/current_style.html#trailing-commas)